config/schedule.rb

```diff
@@ -28,3 +28,12 @@
 every 1.day, at: '4:30 am' do
   rake 'batch:update_daily'
 end
+
+every 1.day, at: '12:30 am' do
+  rake 'cache:warm:homepage'
```

Comment:
Not that much, now. Just top level stats for the default cohort, and only for logged-out people, and the load time is not bad.